flutter_local_notifications/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,10 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.dexterous.flutter_local_notifications_example">
+   <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
+   <uses-permission android:name="android.permission.VIBRATE" />
+   <uses-permission android:name="android.permission.USE_FULL_SCREEN_INTENT" />
+   <uses-permission android:name="android.permission.USE_EXACT_ALARM" />
+   <uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
```

Comment:
Looking at the comment here: https://github.com/abilia/flutter_local_notifications/pull/15/files#diff-523bc26141f04079689f3708d55ee0040abfaaaeaca1fc86039de9326be99885R234-R262

I think we wanna specify these permission in the apps instead.

Reply:
Lol nevermind!